src/OSPSuite.Presentation/UICommands/RunMultipleParameterIdentificationsUICommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using OSPSuite.Core.Domain.ParameterIdentifications;
+using OSPSuite.Core.Domain.Services.ParameterIdentifications;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Core.Services;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OSPSuite.Presentation.UICommands
+{
+    public class RunMultipleParameterIdentificationsUICommand : ActiveObjectUICommand<IReadOnlyList<ParameterIdentification>>
+   {
+      private readonly IParameterIdentificationRunner _parameterIdentificationRunner;
+
+      public RunMultipleParameterIdentificationsUICommand(
+         IParameterIdentificationRunner parameterIdentificationRunner,
+         IActiveSubjectRetriever activeSubjectRetriever)
+         : base(activeSubjectRetriever)
+      {
+         this._parameterIdentificationRunner = parameterIdentificationRunner;
```

Comment:
this?

Reply:
Leftover from temp-code during development. Removed.